tests/run_model_templates_tests.py

```diff
@@ -292,13 +316,18 @@ def run_one_domain_tests_already_in_virtualenv(work_dir, all_tests, verbose):
         raise RuntimeError('The option --run-one-domain-inside-virtual-env may be used only'
                            ' inside the virtual environment of the domain')
 
-    testsuite = unittest.TestSuite()
-    for el in all_tests:
-        testsuite.addTest(el['test'])
-
-    tests_res = run_testsuite(testsuite, work_dir, verbose)
+    if not SHOULD_USE_PYTEST_FOR_TEST_RUNNING:
```

Comment:
please remove this code

Reply:
I'm sorry, Ilya, but this switch may be really useful in some investigations. E.g. for investigation why our tests require such much time I will use the previous unittest runner to have all logs printed second-by-second.

If you wish I could add a command line parameter for this switch.

Reply:
I wish that we have simple, short, clear runner script, without calling itself, without unused code

Reply:
If you add '-s' to the pytest it won't catch outputs

Reply:
> I wish that we have simple, short, clear runner script, without calling itself, without unused code

Me too. 🙂 
But I hope it will be implemented by a person who knows pytest better than we.

>If you add '-s' to the pytest it won't catch outputs
Oh, I see.
Ok, I made change -- removed possibility to use unttest for running the tests.